### ISSUE TITLE
Argument types not being passed into resolvers

### ISSUE THREAD

**Issue by mmanela** [dotansimha/graphql-code-generator#664]
**Describe the bug**
I am very new to all of this so excuse my naiveté. I am using the typescript template on my server side schema to generate the types to use in my resolvers. I checked my schema.json and it is correctly marking my mutations  (or queries) as taking an argument but the generated resolver code types it as any. It does generate the argument type in the file but just doesn't pass it into the **Resolver** interface.

For example, in the below type it does not pass the 4th argument to resolver interface.
```
export type QueryUserResolver<
  R = GqlUser | null,
  Parent = any,
  Context = any
  > = Resolver<R, Parent, Context>;
```

### `Source Type Def
```
  type Query {
    user(id: ID!): User
  }
  type User {
    id: ID!
    username: String!
  }
```

This is how I am calling on the command line:
```
gql-gen --config gqlgen.json --template graphql-codegen-typescript-template --schema _graphql/schema.json --out _graphql/types.ts
```

This is my generator config
```
{
    "generatorConfig": {
      "interfacePrefix": "Gql",
      "noNamespaces": true
    }
  }
``` 

**Expected behavior**
I expect to have the resolver have its arguments typed.

**Environment:**
 - OS: Ubuntu 16 on WSL
 - Codegen: "graphql-code-generator": "^0.12.6", and "apollo-codegen": "^0.20.2",
 - Node: 8.11.2

**Comment by dotansimha** [comment]
@kamilkisiela

**Comment by kamilkisiela** [comment]
It was fixed:
https://github.com/dotansimha/graphql-code-generator/blob/93e13fe6ca8a6c0d065c7911b863e5549c2440ce/packages/plugins/typescript-resolvers/tests/typescript-resolvers.spec.ts#L685-L714

@mmanela @dotansimha I guess it's available since v0.14.0 not in v0.13.0

**Comment by dotansimha** [comment]
Fixed in 0.14.2 🎉
